tools/data-api-sdk/v1/README.md

```diff
@@ -0,0 +1,44 @@
+## `github.com/hashicorp/pandora/tools/data-api-sdk/v1`
+
+This package contains the SDK for the V1 endpoints within [the Data API](../../data-api) - intended to be used by the other Pandora tooling.
+
+The majority of the models used in this SDK come from [the `./models` package](./models) - however this package also contains a handful of models for 1:1 compatibility with the API.
+
+Whilst the SDK contains all the supported endpoints, in the majority of cases the `LoadAllData` method should be sufficient, which can be used like so:
```

Comment:
`LoadAllData` is sufficient as an example here - whilst the raw endpoints are in the SDK they're only going to be used in specialised cases so it's fine to leave these undocumented here, since the methods themselves are documented